spec/sec-left-hand-side-expressions-static-semantics-early-errors-patch.html

```diff
@@ -0,0 +1,9 @@
+<emu-clause id="sec-left-hand-side-expressions-static-semantics-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+  <emu-grammar collapsed><ins>SuperCall : `super` Arguments</ins></emu-grammar>
+  <ul>
+    <li>
+      It is a Syntax Error if the |SuperCall| is nested, directly or indirectly (but not crossing function or `static` initialization block boundaries, excluding arrow function boundaries), within a |ClassStaticBlock|.
```

Comment:
A SuperCall nested indirectly _and_ crossing function boundaries could be invalid, too.
```js
class C {
  static {
    function () { super() }
  }
}
```
We can define the [HasDirectSuper](https://tc39.es/ecma262/#sec-method-definitions-static-semantics-hasdirectsuper) semantics for static block and it is a syntax error if StaticBlock has direct super.

Reply:
`super()` is already invalid in `function`, per https://tc39.es/ecma262/#sec-function-definitions-static-semantics-early-errors:

![image](https://user-images.githubusercontent.com/3902892/98332409-78217e00-1fb3-11eb-8e53-174a7eb4c1f9.png).

That said, I'll investigate using HasDirectSuper similar to what we do for Methods.